pkgs/top-level/all-packages.nix

```diff
@@ -15847,8 +15847,8 @@ in
   mariadb-connector-c = mariadb-connector-c_3_1;
   mariadb-connector-c_3_1 = callPackage ../servers/sql/mariadb/connector-c/3_1.nix { };
 
-  mariadb-galera = mariadb-galera_25;
-  mariadb-galera_25 = callPackage ../servers/sql/mariadb/galera/25.nix {
+  mariadb-galera = mariadb-galera_26;
+  mariadb-galera_26 = callPackage ../servers/sql/mariadb/galera/26.nix {
```

Comment:
Why do we provide `mariadb-galera_*` here?

What's the usecase? If we want to provide users with a way to stick to the old version, then it shouldn't be removed.

If we don't provide this option, there's not much of a point in a "versioned" attribute.

I'd propose to just leave the `mariadb-galera` attribute, and add an alias to `aliases.nix`.

Reply:
Rebased. Only one version left.